docs/application/native/guides/device/battery-monitor.md

```diff
@@ -0,0 +1,199 @@
+# Battery Monitor
+
+You can access the information related to the power consumption by applications or by hardware resources on a battery-powered device for a specific duration of time. The value is returned as the percentage of the total battery capacity.
+
+The power consumption of an application is calculated by accumulating the usage of various individual resources used by the application.
+
+This feature is supported in wearable applications only.
+
+The Battery Monitor APIs provide following provisions to fetch the battery usage:
+
+- Getting battery usage information of an application.
+
+    You can [retrieve battery usage information of an application](#appusage_get) by specifying its application ID, [resource IDs](#resource_key), and the [time duration](#duration_key).
+
+- Getting battery usage information of a resource.
+
+    You can [retrieve battery usage information of single or multiple resources](#resourceusage_get) by specifying its [resource IDs](#resource_key), and the [time duration](#duration_key).
+
+## Prerequisites
+
+To enable your application to use the Battery Monitor functionality:
+
+1. To use the [Battery Monitor](../../api/wearable/latest/group__CAPI__SYSTEM__BATTERY__MONITOR__MODULE.html) API, the application has to request permission by adding the following privileges to the `tizen-manifest.xml` file:
+
+    ```xml
+    <privileges>
+       <privilege>http://tizen.org/privilege/systemmonitor</privilege>
+    </privileges>
+    ```
+
+2. To use the functions and data types of the Battery Monitor API, include the `<battery_monitor.h>` header file in your application:
+
+    ```
+    #include <battery_monitor.h>
+    ```
+
+<a name="appusage_get"></a>
+## Getting Battery Usage Information of Application
+
+To get the battery usage information related to the application ID:
+
+- Get battery usage information of an application for a particular resource over a specific duration of time.
+  To get the resource ID, use the `battery_monitor_resource_id_e` enum and to get the time duration type information, use the `battery_monitor_duration_type_e` enum:
```

Comment:
Thanks @d1-agarwal for your patience and cooperation. 
@mijin85cho thanks for your valuable suggestions. 

@d1-agarwal 
All seems to be fine in the file, but how about inserting a line space between the following lines, I checked the page on staging and the text is in continuity. It would be good if one line space is given.

Line space needed between : 
- 42 & 43 
- 82 & 83
- 102 & 103
- 147 & 148 

Thanks, if you don't agree to my suggestion, then ignore this comment.

Reply:
We can ignore the spacing.
Thanks for the review comments.
@mijin85cho & @nawab-samsung

Reply:
@d1-agarwal 
Thanks